### DISCUSSION TITLE
No detections while using model weights from training custom dataset

### DISCUSSION THREAD

**Discussion by cooldecola** [facebookresearch/detectron2#3579] [Q&A]
Hi, had a custom dataset that I trained using Detectron2 and was happy with the AP value, now I'm trying to use the "model_final.pth" to run inference on a video, however whenever I do this I get no detections on any of the frames. However when I use pretrained weights from detectron2's model zoo it works. Why is this?

This is how I'm loading the weights: 
`cfg.MODEL.WEIGHTS = ("model_final.pth")`

**Comment by PushpakBhoge512** [comment]
I think the syntax is

cfg.MODEL.WEIGHTS = "model_final.pth"

not

cfg.MODEL.WEIGHTS = ("model_final.pth")

**Reply by JNaranjo-Alcazar** [reply]
Did this work?
